schemas/dex-aggregator/SCHEMA.md

```diff
@@ -1,53 +1,66 @@
+| timestamp                | The timestamp of the block this snapshot was taken.       | number |
+| chain_id                 | The standard id of the chain.                             | number |
+| volume_usd               | The volume in USD, in the given snapshot period.          | number |
+| fees_usd                 | The fees (ie, total revenue generated in the protocol) collected in USD, during the snapshot period. | number |
 
 ### Trades
-| Property           | Description                                                                 | Type    |
-|--------------------|-----------------------------------------------------------------------------|---------|
-| timestamp          | The timestamp of the trade.                                                 | number  |
-| chain_id           | The standard id of the chain this trade belongs to.                         | number  |
-| block_number       | The block number in which the trade occurred.                               | number  |
-| transaction_id     | The transaction id associated with this trade.                              | string  |
-| fees               | The amount of fees from this trade (ie, the revenue generated from executing this trade). | number  |
-| fees_usd           | The fees for this trade in USD.                                             | number  |
-| slippage           | (Optional) The slippage of the given trade, as a percentage (ie, 1.3% slippage is 0.013). | number  |
-| user_address       | The address of the user initiating the trade.                               | string  |
-| input_tokens       | The contract address of the input token(s).                                 | [string]|
-| input_token_amounts  | Parallel array to inputTokens, dictating the normalized native amounts of each token traded. | [number]|
-| output_tokens      | The contract address of the output token(s).                                | [string]|
-| output_token_amounts | Parallel array to output, dictating the normalized native amounts of each token traded. | [number]|
-| swap_amount_usd    | The value of the swap in USD.                                               | number  |
+
+List of trades executed on the protocol.
+
+| Property                | Description                                               | Type   |
+|-------------------------|-----------------------------------------------------------|--------|
+| timestamp                | The timestamp of the trade.                               | number |
+| chain_id                 | The standard id of the chain this trade belongs to.       | number |
+| block_number             | The block number in which the trade occurred.             | number |
+| transaction_id           | The transaction id associated with this trade.            | string |
+| fees                     | The amount of fees from this trade (ie, the revenue generated from executing this trade). | number |
```

Comment:
is this token amount? since we are accepting array for input and output, this probably will need to be array as well?

Reply:
We don't really have much external validation on this yet.

Also, the purpose of this PR is not to make edits to the schemas. If you believe things should be changed that are already in `main` this schema should allow help you make a PR and make an update.